client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -179,6 +194,31 @@ const convertToApiPackage = pckg => {
 		'weight',
 		'signature',
 	] );
+	if ( customsItems ) {
+		apiPckg.contents_type =
+			'other' !== pckg.contentsType ? pckg.contentsType || 'merchandise' : pckg.contentsExplanation;
```

Comment:
I noticed that on the server we need to send both `contents_type` and `contents_explanation`. Why aren't we sending both directly from the client in the beginning?

Reply:
Fixed in https://github.com/Automattic/woocommerce-connect-server/pull/1129 and https://github.com/Automattic/wp-calypso/pull/26299